esphome/components/ili9xxx/ili9xxx_display.cpp

```diff
@@ -366,6 +366,10 @@ void ILI9XXXDisplay::init_lcd_() {
 
 // Tell the display controller where we want to draw pixels.
 void ILI9XXXDisplay::set_addr_window_(uint16_t x1, uint16_t y1, uint16_t x2, uint16_t y2) {
+  x1 += this->offset_x_;
+  x2 += this->offset_x_;
+  y1 += this->offset_y_;
+  y2 += this->offset_y_;
```

Comment:
Should that not be: 
```suggestion
  x1 += this->offset_x_;
  x2 -= this->offset_x_;
  y1 += this->offset_y_;
  y2 -= this->offset_y_;
```
Now you only move the display window and go over the display width and height.

Reply:
No, what I have is correct. It's a fixed offset, it has to be applied to all positions.

Reply:
As it is i disapprove this merge, 

There should have be some checks that the address window does not go before 0 and behind width and height.